src/phases/move-effect-phase.ts

```diff
@@ -299,6 +302,16 @@ export class MoveEffectPhase extends PokemonPhase {
            */
           if (lastHit) {
             this.scene.triggerPokemonFormChange(user, SpeciesFormChangePostMoveTrigger);
+            /**
+             * Multi-Lens, Multi Hit move and Parental Bond check for PostDamageAbAttr
+             * other damage source are calculated in damageAndUpdate in pokemon.ts
+             */
+            if (target.hasAbilityWithAttr(PostDamageForceSwitchAbAttr)) {
+              const multiHitModifier = user.getHeldItems().find(m => m instanceof PokemonMultiHitModifier);
+              if (multiHitModifier || user.hasAbilityWithAttr(AddSecondStrikeAbAttr) || move.hasAttr(MultiHitAttr)) {
+                applyPostDamageAbAttrs(PostDamageAbAttr, target, 0, target.hasPassive(), false, [], user);
+              }
+            }
```

Comment:
```suggestion
            if (user.turnData.hitCount > 1) {
              applyPostDamageAbAttrs(PostDamageAbAttr, target, 0, target.hasPassive(), false, [], user);
            }
```
Should be equivalent for a few reasons:
- `hitCount` should have been modified by all the different multi-hit modifiers at this point, either by this `MoveEffectPhase` or a previous `MoveEffectPhase` for the same action.
- `PostDamageAbAttrs` will only apply if the target actually has a post-damage attribute (that isn't suppressed/ignored), and EE is currently the only ability that has as much. This also makes post-damage attributes a bit more scalable.

You might have to fix the imports in this file with this change.